tests/test_sourmash.py

```diff
@@ -72,6 +72,77 @@ def test_sourmash_info_verbose():
     assert "loaded from path" in err
 
 
+def test_load_pathlist_from_file_does_not_exist():
+    from sourmash.sourmash_args import load_pathlist_from_file
```

Comment:
pls move import statement to top of file and remove from all of the functions. No need to run it each time when you can do it at the module level!

Reply:
Sure!